src/main/scala/cromwell/engine/WorkflowActor.scala

```diff
@@ -63,68 +65,75 @@ class WorkflowActor private(binding: WdlBinding, actualInputs: Map[FullyQualifie
   private def notStarted: Receive = {
     LoggingReceive {
 
-      case Start(backend) =>
+      case Start(system, backend) =>
         _primeMover = Option(sender())
-        maybeExecutionStatusStore = Option(new ExecutionStatusStore(binding))
+        maybeExecutionStore = Option(new ExecutionStore(binding))
         maybeSymbolStore = Option(new SymbolStore(binding, actualInputs))
         maybeBackend = Option(backend)
-        executionStatusStore.runnableCalls.foreach { call =>
-          startCallActor(symbolStore, call)
-        }
-        primeMover ! Started
+        // Poll for runnable calls.
+        system.scheduler.schedule(
```

Comment:
That last thought about setting up one's own successor harkens back to the original FSM experiments as the only time new jobs can open up is if something completes.

In general I think we should try to limit unnecessary actor endpoints and unnecessary messages, this has the feeling of both to me